service/dps/storm/common/src/main/java/eu/europeana/cloud/service/dps/storm/utils/XMLSerializer.java

```diff
@@ -0,0 +1,52 @@
+package eu.europeana.cloud.service.dps.storm.utils;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+
+/**
+ * Util class used to serialize object to XML and deserialize XML back to object.
+ *
+ * @param <T> class of the objects
+ */
+public final class XMLSerializer<T> {
```

Comment:
I think this class can be deleted and we should store StatisticsReport object as JSON and not XML.
  Gson gson = new Gson();
  String jsonString = gson.toJson(statisticsReportObject); //This should serialize the object
 StatisticsReport statisticsReport = gson.fromJson(jsonString, StatisticsReport.class); //This should deserialize the object.